apps/product-console/.env

```
@@ -0,0 +1,66 @@
+
+# Matcher Configuration
+MATCHER_BASE_PATH='http://matcher:8080/api/v1'
+
+# MongoDB Configuration
+MONGODB_URI='mongodb://${MONGODB_USER}:${MONGODB_PASS}@midaz-mongodb:5703/?directConnection=true'
+MONGODB_DB_NAME='midaz-console'
+MONGODB_USER='midaz'
+MONGODB_PASS='your-mongodb-password'
+
+# General Configuration
+NEXT_PUBLIC_MIDAZ_CONSOLE_AVATAR_ALLOWED_FORMAT='jpeg,jpg,png,svg+xml'
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)